big_o_exercise/readme.md

```diff
@@ -4,16 +4,16 @@
 
 Simplify the following big O expressions as much as possible:
 
-1. `O(n + 10)`
-2. `O(100 * n)`
-3. `O(25)`
-4. `O(n^2 + n^3)`
-5. `O(n + n + n + n)`
-6. `O(1000 * log(n) + n)`
-7. `O(1000 * n * log(n) + n)`
-8. `O(2^n + n^2)`
-9. `O(5 + 3 + 1)`
-10. `O(n + n^(1/2) + n^2 + n * log(n)^10)`
+1. `O(n + 10)` // O(n)
+2. `O(100 * n)` // O(n)
+3. `O(25)` // O(1)
+4. `O(n^2 + n^3)` // O(n^2 )
+5. `O(n + n + n + n)` // O(n)
+6. `O(1000 * log(n) + n)` // O(log(n))
+7. `O(1000 * n * log(n) + n)` // O(n)
```

Comment:
Thanks, 
For some reason I kept thinking 
O(n) > O(nlog(n))> O(log(n)) which is why my answers were consistently wrong